packages/core/test/render3/instructions/styling_spec.ts

```diff
@@ -403,51 +403,51 @@ describe('styling', () => {
 
   describe('toStylingArray', () => {
     describe('falsy', () => {
-      it('should return empty ArrayMap', () => {
-        expect(toStylingArrayMap(arrayMapSet, null !, '')).toEqual([] as any);
-        expect(toStylingArrayMap(arrayMapSet, null !, null)).toEqual([] as any);
-        expect(toStylingArrayMap(arrayMapSet, null !, undefined)).toEqual([] as any);
-        expect(toStylingArrayMap(arrayMapSet, null !, [])).toEqual([] as any);
-        expect(toStylingArrayMap(arrayMapSet, null !, {})).toEqual([] as any);
+      it('should return empty KeyValueArray', () => {
+        expect(toStylingArrayMap(keyValueArraySet, null !, '')).toEqual([] as any);
```

Comment:
Seems like we need to rename `toStylingArrayMap` as well

```suggestion
        expect(toStylingKeyValueArray(keyValueArraySet, null !, '')).toEqual([] as any);
```